.swagger-codegen-ignore

```diff
@@ -23,7 +23,11 @@
 #!docs/README.md
 README.md
 Gemfile
+app/models/*.rb
```

Comment:
@syncrou Should this be specific model files that have changes in them.
Wouldn't we want the codegen to build new ones for us.

Reply:
Yeah, my thought was that code would already be built and modified but I agree with your point. I'll add specific models with the understanding that we manage this file as new 'official' models come online.